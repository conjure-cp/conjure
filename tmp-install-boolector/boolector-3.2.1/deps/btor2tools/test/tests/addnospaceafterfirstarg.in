1 sort bitvec 32
2 zero 1
3 add 1 2nospace
