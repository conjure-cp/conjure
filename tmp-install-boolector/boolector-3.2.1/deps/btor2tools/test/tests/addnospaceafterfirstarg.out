*** catbtor: parse error in 'addnospaceafterfirstarg.in' line 3: expected space after argument (argument missing)
